setup.cfg

```diff
@@ -1,7 +1,8 @@
 [tool:pytest]
-norecursedirs = .git venv build dist *egg
-addopts = -rfEsxwX --cov readchar
+testpaths = tests
+addopts = -rfEsxwX -s --cov=readchar
 
 [flake8]
+max-line-length=88
```

Comment:
please, do you mind modifying the file .pre-commit-config.yaml to set the same configuration for black?

Reply:
will do!

Reply:
to clerify this: a linelength of 88 ist the default for black, so its doesn't need to be set explicitly. But it makes sense to have flake8 check for the smae line length.